testing/ostest/ostest_main.c

```diff
@@ -581,8 +585,16 @@ static int user_main(int argc, char *argv[])
 #endif
 
 #if defined(CONFIG_ARCH_HAVE_FORK) && defined(CONFIG_SCHED_WAITPID)
+#ifndef CONFIG_BUILD_KERNEL
       printf("\nuser_main: vfork() test\n");
       vfork_test();
```

Comment:
vfork should support too?

Reply:
vfork->arm_fork do not support in CONFIG_BUILD_KERNEL, and will hang for some reason, there will be much effect to support it.

Reply:
Ok.